apps/src/templates/manageStudents/ManageStudentsLoginInfo.jsx

```diff
@@ -124,24 +166,37 @@ class ManageStudentsLoginInfo extends Component {
           <div>
             <p>{i18n.setUpClassEmailIntro()}</p>
             <SafeMarkdown
-              markdown={i18n.setUpClassEmail1({
-                createAccountLink: `${studioUrlPrefix}/users/sign_up`,
-              })}
+              markdown={i18n
+                .setUpClassEmail1({
+                  createAccountLink: `${studioUrlPrefix}/users/sign_up`,
+                })
+                .replace(/^\d./, `${++counter}.`)}
             />
             <SafeMarkdown
-              markdown={i18n.setUpClassEmail2({
-                joinLink: `${studioUrlPrefix}/join/${sectionCode}`,
-              })}
+              markdown={i18n
+                .setUpClassEmail2({
+                  joinLink: `${studioUrlPrefix}/join/${sectionCode}`,
+                })
+                .replace(/^\d./, `${++counter}.`)}
+            />
+            <GlobalRegionWrapper
+              component={() => (
+                <SafeMarkdown
+                  markdown={i18n
+                    .setUpClass3({
+                      parentLetterLink: teacherDashboardUrl(
+                        sectionId,
+                        '/parent_letter'
+                      ),
+                    })
+                    .replace(/^\d./, `${++counter}.`)}
+                />
+              )}
+              componentId="ClassroomSetUpStep3"
```

Comment:
More rewriting the step numbers since some intermediate steps are removed in some regions.